### DISCUSSION TITLE
nbb logo

### DISCUSSION THREAD

**Discussion by borkdude** [babashka/nbb#90] [Ideas]
@MnRA You already have some ideas, can you post them here as separate posts? Then we can discuss in the comments of each.

**Comment by MrnoRac** [comment]
Sure, here's the latest version as PNG, because github doesn't like svgs apparently
![nbb-small](https://user-images.githubusercontent.com/79454885/137729455-26bb5ab2-42d6-498a-af49-303c90781311.png)
.

**Reply by MrnoRac** [reply]
yeah I exported in too high of a dpi, woops!

**Reply by MrnoRac** [reply]
could consider making the lambdas into JS, or even just changing the glasses into the node hexagon shapes instead of the head.

**Reply by borkdude** [reply]
Yeah, worth a shot! Please save your alternatives so we can revert to previous ideas too.

**Reply by borkdude** [reply]
OK, let's have a shot at playing with the content of the glasses.

**Reply by MrnoRac** [reply]
![image](https://user-images.githubusercontent.com/79454885/137799163-ea80f227-8647-4fff-ba63-dd4fa936e6a3.png)
![image](https://user-images.githubusercontent.com/79454885/137799404-0d9b9a7d-d5fd-4162-a981-8f0f4ebfeda2.png)
simple idea, kinda breaks the lambda spirit, but at least it makes it obvious it's js based

**Reply by borkdude** [reply]
Can you try one with only `>_` or so? This was what we had early on in the bb logo but I managed to convince Nikitonsky that a lambda would be cooler. However, if you type node you get this:

```
$ node
Welcome to Node.js v14.18.0.
Type ".help" for more information.
>
```
And having a simple bigger thing instead of smaller works better with small emoticons too perhaps?

**Reply by MrnoRac** [reply]
fun fact that JS is the same exact font from this JS logo:
![image](https://user-images.githubusercontent.com/79454885/137799942-a1cec9ef-a660-49ba-bc8a-8b0bc88b7910.png)

**Reply by MrnoRac** [reply]
![image](https://user-images.githubusercontent.com/79454885/137800304-6bc745a7-7828-4566-9c9a-2788dd5c2f77.png)

**Reply by MrnoRac** [reply]
excuse the blurriness, these are just screenshots, because exporting properly is tedious 😄

**Reply by borkdude** [reply]
OK, I'll say: let's stick with the original you posted here: https://github.com/borkdude/nbb/discussions/90#discussioncomment-1495172

Unless you have some other cool ideas. And then you could make a PR similar to what's in the babashka repo under logo?

Thanks very much!

**Reply by borkdude** [reply]
@MrnoRac Perhaps for fun we could try a different hair color (purple) or skin tone combination.

**Reply by borkdude** [reply]
But I'm also fine with leaving it here and staying close to the original: less details is probably better.

**Reply by MrnoRac** [reply]
Cool I'll open the PR soonish, I want to do this from my other computer which I haven't had time to set up yet so it doesn't mess with my work stuff.

**Comment by borkdude** [comment]
@MrnoRac I think it would be worth trying a version without sunglasses with the cute eyes from your work repository.
But with the hexagon-shaped head and node color. Perhaps try to make the scarve also a bit hexagonish?

**Reply by MrnoRac** [reply]
Here's a few variations
![image](https://user-images.githubusercontent.com/79454885/137794778-659c8cf0-bdf5-4421-a71e-0f01fbd82763.png)
![image](https://user-images.githubusercontent.com/79454885/137794856-7428b040-8e43-4079-a0ba-c8b9381e420e.png)
![image](https://user-images.githubusercontent.com/79454885/137795580-2a8d06be-fcef-4e75-a11f-0ebb41fb0b3d.png)

**Reply by MrnoRac** [reply]
Though keeping it close to the original is probably a good thing because they are very similar, but distinct!

**Reply by borkdude** [reply]
I agree. And I like the closer to the original better now that I see this. Thank you!

**Reply by jaidetree** [reply]
Also agree. Makes it seem more like they're sisters in the same family 😆 The hexagonal face shape though is a really nice touch I didn't fully appreciate until now.
